npm/cloudbuild.yaml

```diff
@@ -78,3 +96,5 @@ images:
 - 'gcr.io/$PROJECT_ID/npm:node-8.4.0'
 - 'gcr.io/$PROJECT_ID/npm:node-9.11.2'
 - 'gcr.io/$PROJECT_ID/npm:node-10.10.0'
```

Comment:
Please add `gcr.io/$PROJECT_ID/npm:lts` to the list of pushed images.

Reply:
@bendory npm:lts tag added

Thank you!